src/EvoSC.Common/Services/PlayerManagerService.cs

```diff
@@ -39,25 +36,23 @@ public async Task<IPlayer> GetOrCreatePlayerAsync(string accountId)
         return await CreatePlayerAsync(accountId);
     }
 
-    public async Task<IPlayer> CreatePlayerAsync(string accountId)
+    public Task<IPlayer> CreatePlayerAsync(string accountId) => CreatePlayerAsync(accountId, null);
+
+    public async Task<IPlayer> CreatePlayerAsync(string accountId, string? name)
     {
         var playerLogin = PlayerUtils.ConvertAccountIdToLogin(accountId);
 
         TmPlayerDetailedInfo? playerInfo = null;
-        // TODO: Create player with default properties when limited information is available #81 https://github.com/EvoTM/EvoSC-sharp/issues/81
         try
         {
             playerInfo = await _server.Remote.GetDetailedPlayerInfoAsync(playerLogin);
         }
         catch (Exception ex)
         {
-            _logger.LogDebug(ex, "Player not on server");
+            // ignore the error because we will then set the name instead
```

Comment:
Think it could be a good idea to still LogDebug the action, so that if you run this while in debug mode, you can see the exception being thrown in the log (it wont affect the rest of the code, the exception is still ignored as it's not thrown further out of the scope).

Reply:
This should probably be trace in this case because this is not actually an error, just purely informational and can cause unnecessary logging output.